variable_set_test.go

```diff
@@ -349,6 +398,109 @@ func TestVariableSetsApplyToAndRemoveFromWorkspaces(t *testing.T) {
 	})
 }
 
+func TestVariableSetsApplyToAndRemoveFromProjects(t *testing.T) {
+	// TO DO: Remove when TFE GA
+	skipUnlessBeta(t)
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
```

Comment:
💅 There's a mix of `defer` and `t.Cleanup` in the new code. Both work, but I was wondering if you ran into an issue with t.Cleanup, because that's what we prefer these days.